### ISSUE TITLE
run-lock-<hash> file not found

### ISSUE THREAD

**Issue by kkebb** [phoronix-test-suite/phoronix-test-suite#132]
When I startup the phoromatic server, I see the following NOTICE:
`[NOTICE] unlink(/var/lib/phoronix-test-suite/run-lock-RLHsTW): No such file or directory in pts_file_io:38`

however, when I `ls -lah` I get the following contents:

```
[root@d09cd95401a8 /]# ls -lah /var/lib/phoronix-test-suite/
total 56K
drwxr-xr-x 11 root root 4.0K Oct  6 14:48 .
drwxr-xr-x 14 root root 4.0K Oct  4 21:08 ..
-rw-------  1 root root 1.7K Oct  6 14:48 core.pt2so
drwxr-xr-x  2 root root 4.0K Oct  4 21:08 download-cache
-rw-r--r--  1 root root  955 Oct  6 14:48 graph-config.json
drwxr-xr-x  2 root root 4.0K Oct  4 20:56 installed-tests
drwxr-xr-x  2 root root 4.0K Oct  4 20:56 modules
drwxr-xr-x  2 root root 4.0K Oct  4 20:56 modules-data
drwxr-xr-x  3 root root 4.0K Oct  6 14:48 phoromatic
-rw-r--r--  1 root root    0 Oct  6 14:48 run-lock-p3A26T
-rw-------  1 root root 2.8K Oct  4 20:56 temp.pt2so
drwxr-xr-x  5 root root 4.0K Oct  4 21:08 test-profiles
drwxr-xr-x  2 root root 4.0K Oct  4 20:56 test-results
drwxr-xr-x  5 root root 4.0K Oct  4 21:09 test-suites
drwxr-xr-x  2 root root 4.0K Oct  4 21:08 xsl
```

note that the contents above are displaying `run-lock-p3A26T` (note that it is 0 kb size file) instead of `run-lock-RLHsTW` which is what the server is searching for in the NOTICE message above.

Here is the file contents for `run-lock-p3A26T`:

```
[root@d09cd95401a8 /]# cat /var/lib/phoronix-test-suite/run-lock-p3A26T
[root@d09cd95401a8 /]#
```

**Comment by michaellarabel** [comment]
That notice can be safely ignored and isn't causing an actual issue, but the run-lock files will be cleaned up and better handled in hopefully the next release.

**Comment by kkebb** [comment]
Is there an issue tracking this so I can follow up with it?

**Comment by michaellarabel** [comment]
I'm not aware of any other tickets on PTS GitHub pertaining to the run-lock file handling, but can post here when I get around to working on it to clean up that notice.

**Comment by kkebb** [comment]
Sounds good - much appreciated!

**Comment by michaellarabel** [comment]
The run-lock clean-up had landed in PTS 7.0. Feel free to open new issue if you run into any similar issue.
